# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: cc4de7b25404
# Full Hash: cc4de7b25404f927d2219f81000d0e0f5c9f7cfe
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p4 - Move nsPrintData::mNumPrintablePages to nsPrintJob. r=emilio
#   
#   Depends on D146975
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146976
# ==============================================================================

diff -r f946921b9173 -r cc4de7b25404 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
+++ b/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
@@ -266,6 +266,8 @@
 
   float mScreenDPI = 115.0f;
 
+  int32_t mNumPrintablePages = 0;
+
   bool mCreatedForPrintPreview = false;
   bool mIsCreatingPrintPreview = false;
   bool mIsDoingPrinting = false;
